support/support.go

```diff
@@ -203,7 +209,7 @@ func Run(ctx context.Context, d *Deps) (*Bundle, error) {
 
 	// Ensure we capture logs from the client.
 	var logw strings.Builder
-	d.Log.AppendSinks(sloghuman.Sink(&logw))
+	d.Log = d.Log.AppendSinks(sloghuman.Sink(&logw))
```

Comment:
How are these forms different, append doesn't mutate?

Reply:
Yeah that was my observation.